Sources/Public/Animation/AnimationView.swift

```diff
@@ -685,7 +685,7 @@ final public class AnimationView: LottieView {
   var animationLayer: RootAnimationLayer? = nil
 
   /// Set animation name from Interface Builder
-  @IBInspectable var animationName: String? {
```

Comment:
One issue with this API is that `animationName` and `animation` can become out of sync. That is:

```swift
animationView.animationName = "Animation A"
animationView.animation = .named("AnimationB")
print(animationView.animationName) // incorrectly prints "Animation A"
```

Since you can get the equivalent behavior by just setting `animation = Animation.named(...)`, I think we should leave this internal.